python/oneflow/test/modules/test_sort.py

```diff
@@ -84,12 +84,29 @@ def test_sort_with_random_data(test_case):
         y = torch.sort(x, dim=random(low=-4, high=4).to(int), descending=random_bool())
         return y[0], y[1]
 
+    @autotest(n=5, auto_backward=False, check_graph=True)
```

Comment:
```suggestion
    @autotest(n=5, auto_backward=False)
```
check_graph 默认是 True，不过这个不影响什么，看你个人风格，下同